test/e2e/lib/video-recorder.js

```diff
@@ -69,7 +55,7 @@ export function startVideo() {
 	const dateTime = new Date().toISOString().split( '.' )[ 0 ].replace( /:/g, '-' );
 	const fileName = `${ global.displayNum }-${ dateTime }.mpg`;
 	file = path.resolve( path.join( './screenshots/videos', fileName ) );
-	createDir( path.dirname( file ) );
+	fs.mkdirSync( path.dirname( file ), { recursive: true } );
```

Comment:
From the docs (https://nodejs.org/api/fs.html#fs_fs_mkdir_path_options_callback):

> a `recursive` property indicating whether parent directories should be created. Calling `fs.mkdir()` when path is a directory that exists results in an error only when `recursive` is false.